tasks/main.yml

```diff
@@ -16,6 +16,8 @@
   ansible.builtin.include_tasks: ./websrv_install.yml
 
 - name: Install certificates
+  tags:
```

Comment:
Missing documentation about this tag in README.

Reply:
Same as above. I'll update RREADME once the Pull Request is accepted. As stated, README needs quite some further updates.

Reply:
Understood. I'm just in the position that my changes started outside a git repository and only afterwards decided that it might be goot to contribute. Now I run into the issue that I need to track the changes in the git-repository manually. Getting this quickly in safes therefore a lot of effort on my end. Further, I do have quite some suggestions for README, which might need some debate. I installed and used your great ansible-role and trapped into all issues as a newby to ansible with the provided documentation, thus all questions that are not yet in the documentation. Sur I can separate this and only add the minimum. Please advice if minimum documention is what you look for.

Reply:
> ... and only afterwards decided that it might be good to contribute.

And for that we thank you 😃 

> Further, I do have quite some suggestions for README, which might need some debate

Then let's do it in an other Pull request. the smaller the PR the quickest it is reviewed 😃 

> Please advice if minimum documention is what you look for.

I'm looking for the doc to explain to a user how to use any aspect of the role **at any moment/commit**, _in main_.

So any new variable that add feature should be documented in the `defaults/main.yml` with comments and use cases should be described in the readme, with examples if possible.
To write it easier, my recommandations are:
  - Don't write the doc by assuming your next suggestions
  - go for simple, bare examples
  - If porting your changes to the readme is too much work, maybe starting over the lastest version in your fork will be simplier. You might even be happier with the "new" version of your doc. 😃

Reply:
Done. I've kept my changes to minimu.